README.md

```diff
@@ -6,45 +6,69 @@ A Floating UI wrapper for Svelte.
 
 This project is maintained by [Hugo Korte](https://github.com/Hugos68), [Skeleton Labs](https://www.skeletonlabs.co/), and the [Svelte](https://svelte.dev/) community. Based on the amazing work by [Floating UI](https://github.com/floating-ui/floating-ui).
 
-## Contributing
+## Installation
 
-### Developing
-
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Install the package using your preferred package manager.
 
 ```bash
-npm run dev
-
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
+npm add @skeletonlabs/floating-ui-svelte
+# pnpm add @skeletonlabs/floating-ui-svelte
+# yarn add @skeletonlabs/floating-ui-svelte
+# bun add @skeletonlabs/floating-ui-svelte
 ```
 
-Everything inside `src/lib` is part of your library, everything inside `src/routes` can be used as a showcase or preview app.
-
-### Building
+## Usage
 
-To build your library:
+Import the desired hook or component from floating-ui-svelte, for example:
 
-```bash
-npm run package
-```
+```js
+import { useFloating, type UseFloatingOptions } from '@skeletonlabs/floating-ui-svelte';
 
-To create a production version of your showcase app:
+const options: UseFloatingOptions = { ... };
 
-```bash
-npm run build
+const floating = useFloating(options);
```

Comment:
Not sure how much is implemented right now, but I think it would be useful to have a more complete example. Like, what the user has to do with `floating`? How to tell Floating UI which element to "float"?

Reply:
We'll be keeping the readme as an API reference. The website will have practical examples.

Reply:
I do think Sebas still has a point, a barebones *working* example is probably a good thing to add. Examples should be in addition to get specific results. The docs should still make it clear how to use the hook.

Reply:
I've added a new barebones example. Let me know if that clears it up.